docs/en/ufs/S3.md

```diff
@@ -279,44 +249,92 @@ alluxio.underfs.s3.upload.threads.max=80
 alluxio.underfs.object.store.service.threads=80
 ```
 
-## S3 Access Control
+## Identity and Access Control of S3 Objects
+
+[S3 identity and access management](https://docs.aws.amazon.com/AmazonS3/latest/dev/s3-access-control.html)
+is very different from the security model of a UNIX-like file system
+(e.g., S3 ACL has no groups and no directory-level setting).
+Alluxio makes a best effort to inherit file owner and permission bits from S3 ACL information.
+Alluxio master loads the ACL information of objects and buckets on the first time when their
+metadata is requested, then translates it to Alluxio permission and store the metadata information
+as a part of Alluxio filesystem metadata.
+
+### Why is 403 Access Denied Error Returned
+
+The S3 credentials set in Alluxio configuration corresponds to an AWS user.
+If this user does not have the required permissions to access an S3 bucket or object,
+a 403 permission denied error will be returned.
 
-If Alluxio security is enabled, Alluxio enforces the access control inherited from underlying object
-storage.
+If you see 403 error in Alluxio server log when accessing a S3 service, you should double check
+1. you are using the correct credentials
+1. your AWS user does have the permission to access the buckets and objects mounted to Alluxio
```

Comment:
`Your AWS user has permissions to access the buckets and objects mounted to Alluxio.`